src/main/java/funbox/Main.java

```diff
@@ -0,0 +1,38 @@
+package funbox;
+
+import java.io.IOException;
+
+import funbox.FunBox;
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+
+    private FunBox funBox = new FunBox();
+
+    /**
+     * Display the GUI for the user.
```

Comment:
I believe u have to use the verb form, as per the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments):

> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)

```suggestion
     * Displays the GUI for the user.
```